model/src/main/java/org/projectnessie/api/TreeApi.java

```diff
@@ -233,7 +233,19 @@ LogResponse getCommitLog(
           Integer maxRecords,
       @Parameter(description = "pagination continuation token, as returned in the previous LogResponse.token")
       @QueryParam("pageToken")
-          String pageToken)
+          String pageToken,
+      @Parameter(description = "The author of a commit. This is the original committer")
+      @QueryParam("author")
+      String author,
+      @Parameter(description = "The entity that performed the commit")
```

Comment:
Nit: "entity" feels a bit too abstract. Maybe "person or system"?

Reply:
I "stole" those descriptions from `CommitMeta` but I agree we should improve them. Will follow up shortly